charts/kyverno-policies/templates/istio/proxy-quit/cluster-policy-create-role.yaml

```diff
@@ -15,10 +16,10 @@ metadata:
     policies.kyverno.io/description: >-
       In order for jobs within an istio service mesh to successfully complete
       there needs to be a way to quit the istio-proxy sidecar. This policy
-      watches all namespaces and watches for job based pods admissions. 
-      From there a narrow scoped RBAC role and rb that are attached to the pod's 
+      watches all namespaces and watches for job based pods admissions.
+      From there a narrow scoped RBAC role and rb that are attached to the pod's
       service account to allow for "get", "list", "watch" on all containers on said pod.
```

Comment:
regression 'bug' from previous. This should just read `to allow for "get" on all ....`